docker_templates/templates/docker_images/ros2_entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+# setup ros2 environment
+source "/opt/ros/$ROS2_DISTRO/setup.bash"
```

Comment:
Its simple enough to have separate entry points. Also, if you stack the dockerfile, then at least the entry point files wouldn't collide.